zaas-service/src/main/java/org/zowe/apiml/zaas/security/service/JwtUtils.java

```diff
@@ -10,53 +10,66 @@
 
 package org.zowe.apiml.zaas.security.service;
 
-import io.jsonwebtoken.Claims;
-import io.jsonwebtoken.ExpiredJwtException;
-import io.jsonwebtoken.JwtException;
-import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.*;
+import io.jsonwebtoken.io.Deserializer;
+import io.jsonwebtoken.jackson.io.JacksonDeserializer;
 import lombok.experimental.UtilityClass;
 import lombok.extern.slf4j.Slf4j;
 import org.zowe.apiml.security.common.token.TokenExpireException;
 import org.zowe.apiml.security.common.token.TokenNotValidException;
 
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.Map;
+
 @Slf4j
 @UtilityClass
 public class JwtUtils {
 
+    private static final String HEADER_NONE_SIGNATURE = Base64.getEncoder().encodeToString("""
+        {"typ":"JWT","alg":"none"}""".getBytes(StandardCharsets.UTF_8));
+
+    private static final Deserializer<Map<String, ?>> JACKSON_DESERIALIZER = new JacksonDeserializer<>();
     private static final String TOKEN_IS_NOT_VALID_DUE_TO = "Token is not valid due to: {}.";
 
+    /**
+     * The method just read the claims without any validation of token itself. It could be use just in the case if
+     * validity is check in the related code.
+     *
+     * @param jwt token to be parsed
+     * @return parsed claims or empty object if the jwt is null
+     * @throws TokenNotValidException in case of invalid input, or TokenExpireException if JWT is expired
+     */
```

Comment:
```suggestion
    /**
     * This method reads the claims without validating the token signature. It should be used only if the validity was checked in the calling code.
     *
     * @param jwt token to be parsed
     * @return parsed claims or empty object if the jwt is null
     * @throws TokenNotValidException in case of invalid input, or TokenExpireException if JWT is expired
     */
```